codebuild/cd/publish-to-prod-pypi.yml

```diff
@@ -8,11 +12,13 @@ phases:
       - export PATH=$PATH:$HOME/.local/bin
       - python3 -m pip install --user --upgrade pip
       - python3 -m pip install --user --upgrade twine setuptools wheel awscli PyOpenSSL six
+      - echo "\nBuild version data:"
+      - echo "\nPython Version:"; python3 --version
   pre_build:
     commands:
-      - cd aws-iot-device-sdk-python-v2
-      - pypirc=$(aws secretsmanager get-secret-value --secret-id "prod/aws-sdk-python-v2/.pypirc" --query "SecretString" | cut -f2 -d\") && echo "$pypirc" > ~/.pypirc
-      - export PKG_VERSION=$(git describe --tags | cut -f2 -dv)
+      - cd $CODEBUILD_SRC_DIR/aws-iot-device-sdk-python-v2
+      - pypirc=$(aws secretsmanager get-secret-value --secret-id "cd/aws-sdk-python-v2-prod/.pypirc" --query "SecretString" | cut -f2 -d\") && echo "$pypirc" > ~/.pypirc
+      - export PKG_VERSION=$(cat VERSION)
       - echo "Updating package version to ${PKG_VERSION}"
       - sed --in-place -E "s/__version__ = '.+'/__version__ = '${PKG_VERSION}'/" awsiot/__init__.py
```

Comment:
Instead of storing the version in 2 files: `VERSION` and `__init__.py`
you can get away with storing it in only 1: `__init__.py`

then you can call
```
$ python3 setup.py version
1.11.9
```

which is just as easy as calling:
```
$ cat VERSION
1.11.9
```

of course, this specific trick only works for python, so if you were gunning for 100% consistency across repos, you might not want to use it

Reply:
Hmm, that is not a bad idea. We're also using `VERSION` to trigger the CD pipeline in Codebuild, but I suppose we could upload the `__init__.py` file though and use that instead. Javascript we can use the `package.json` and Maven we could, in theory, use the `pom.xml`. It would be harder to parse for Javascript and Java though.

I'm torn as I like the consistency of using VERSION in all 4 SDKs, but on the other hand, the data is already stored in the Python `__init__.py` file, so there is some data duplication. Likewise, the Javascript `package.json` and Java `pom.xml` will be in similar states.
Will think further on this 👍